lib/builder/builder.js

```diff
@@ -633,6 +643,10 @@ export default class Builder {
       this.customFilesWatcher.close()
     }
 
+    if (this.nuxtConfigWatcher) {
```

Comment:
Can't we reuse `customPatterns`, is there any special reason for a separated watcher ?